lib/apiservers/engine/backends/system.go

```diff
@@ -262,6 +265,25 @@ func getImageCount(plClient *client.PortLayer) int {
 	return len(images)
 }
 
+// Use the Portlayer's support for docker ps to get the container count
 func getContainerStatus(plClient *client.PortLayer) (ContainerStatus, error) {
-	return ContainerStatus{0, 0, 0, 0}, nil
+	var status ContainerStatus
+
+	all := new(bool)
+	*all = true
+	containList, err := plClient.Containers.GetContainerList(containers.NewGetContainerListParams().WithAll(all))
```

Comment:
Minor nit: Any reason we can't go the extra inch and call this `containerList`? :)